job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -378,6 +378,8 @@ definitions:
       description:
         type: string
         description: The description of the job
+      expirationPolicy:
```

Comment:
Should we shorten this to `expiry`?  (And you can put the definition back to `expirationPolicy` if that makes more sense.)